shared/team-building/index.tsx

```diff
@@ -132,15 +138,178 @@ const ContactsBanner = ({onRedoSearch, onRedoRecs}: {onRedoSearch: () => void; o
 }
 
 class TeamBuilding extends React.PureComponent<Props, {}> {
+  sectionListRef = React.createRef<Kb.SectionList>()
   componentDidMount = () => {
     this.props.fetchUserRecs()
   }
 
+  _alphabetIndex = () => {
+    return (
+      <Kb.Box2 direction="vertical" centerChildren={true} style={styles.alphabetIndex}>
+        {this.props.recommendations &&
+          this.props.recommendations.map(section =>
+            section.label && section.label.length <= 3 ? (
```

Comment:
Why the `section.label.length` check? And how can it be empty? Above you do a `.filter(Boolean)` so empty ones shouldn't appear in the list at all.

Reply:
I updated the code to check `=== 1` and do the last section "0-9" separately. I'll remove the empty check